packages/authentication/src/__tests__/acceptance/basic-auth.acceptance.ts

```diff
@@ -3,37 +3,37 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+import {inject, Provider, ValueOrPromise} from '@loopback/context';
 import {Application} from '@loopback/core';
+import {anOpenApiSpec} from '@loopback/openapi-spec-builder';
+import {api, get} from '@loopback/openapi-v3';
 import {
-  RestBindings,
-  ParseParams,
   FindRoute,
   InvokeMethod,
-  Send,
+  ParseParams,
   Reject,
-  SequenceHandler,
-  RestServer,
-  RestComponent,
   RequestContext,
+  RestBindings,
+  RestComponent,
+  RestServer,
+  Send,
+  SequenceHandler,
 } from '@loopback/rest';
-import {api, get} from '@loopback/openapi-v3';
 import {Client, createClientForHandler} from '@loopback/testlab';
-import {anOpenApiSpec} from '@loopback/openapi-spec-builder';
-import {inject, Provider, ValueOrPromise} from '@loopback/context';
+import {Strategy} from 'passport';
+import {BasicStrategy} from 'passport-http';
 import {
   authenticate,
-  UserProfile,
-  AuthenticationBindings,
   AuthenticateFn,
-  AuthenticationMetadata,
+  AuthenticationBindings,
   AuthenticationComponent,
+  AuthenticationMetadata,
+  UserProfile,
 } from '../..';
-import {Strategy} from 'passport';
-import {BasicStrategy} from 'passport-http';
 
 const SequenceActions = RestBindings.SequenceActions;
 
-describe('Basic Authentication', () => {
+describe.skip('Basic Authentication', () => {
```

Comment:
I only add '.skip(' to the test file, but it also resorted the imports. Not sure how to prevent that.
It shouldn't matter, @jannyHou's PR, https://github.com/strongloop/loopback-next/pull/2746, needs to rewrite this test file.

Reply:
It's ok, should be some auto fix by prettier/tslint